tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -151,3 +165,97 @@ Get Invalid VCH
 
 Get Invalid VCH Within Datacenter
     Get Path Under Target  /datacenter/${DC-ID}/vch/INVALID
+    
+Get Deleted VCH
+    Create VCH    '{"name":"%{VCH-NAME}-api-test-simple","compute":{"resource":{"name":"%{TEST_RESOURCE}"}},"storage":{"image_stores":["ds://%{TEST_DATASTORE}"]},"network":{"bridge":{"ip_range":"172.16.0.0/12","port_group":{"name":"%{BRIDGE_NETWORK}"}},"public":{"port_group":{"name":"${PUBLIC_NETWORK}"}}},"auth":{"server":{"generate":{"cname":"vch.example.com","organization":["VMware, Inc."],"size":{"value":2048,"units":"bits"}}},"client":{"no_tls_verify": true}}}'
+    Verify Return Code
+    Verify Status Created
+          
+    ${expectedId}=  Get VCH ID  %{VCH-NAME}-api-test-simple
+    Get Path Under Target  vch/${expectedId}
```

Comment:
If we create a new VCH and inspect this using API, do we want to check the output of `Get Path Under Target` and status code? Or we could skip this step and only check `Not Found` after deleting the VCH

Reply:
will check return code after Get